fmn/consumer/backends/sse.py

```diff
@@ -0,0 +1,177 @@
+import json
+import logging
+import uuid
+
+import datetime
+import pika
+
+import six
+
+from fmn.consumer.backends.base import BaseBackend, shorten
+import fedmsg.meta
+
+import sys
+reload(sys)
+sys.setdefaultencoding('utf-8')
```

Comment:
when i getting messages from fedmsg it was throwing utf8 errors. I can remove this and test